app/controllers/application_controller.rb

```diff
@@ -360,7 +360,7 @@ def textify_new_pubs(author)
     author.each do |genre|
       next unless genre[1].class == Array # skip the :latest key
 
-      worksbuf = "<strong>#{I18n.t(genre[0])}:</strong> "
+      worksbuf = "<strong>#{textify_genre(genre[0])}:</strong> "
```

Comment:
Here and below: we should not use `I18n.t(genre)`, we should use `textify_genre` helper instead.